samples/bluetooth/central_hids/README.rst

```diff
@@ -40,10 +40,14 @@ Button 1:
    Toggle the CAPSLOCK LED on the connected keyboard using Write without response.
    This function is available only if the connected keyboard is set to work in Boot Protocol Mode.
 
+   When pairing, press this button to confirm the passkey value that is printed on the COM listener to pair with the other device.
+
 Button 2:
    Switch between Boot Protocol Mode and Report Protocol Mode.
    This function is available only if the connected peer supports the Protocol Mode Characteristic.
 
+   When pairing, press this button to reject the passkey value that is printed on the COM listener to prevent pairing with the other device.
```

Comment:
```suggestion
   When pairing, press the button to reject the passkey value that is printed on the COM listener to prevent pairing with the other device.
```